docs/connector-development/connector-builder-ui/incremental-sync.md

```diff
@@ -0,0 +1,133 @@
+# Incremental sync
+
+An incremental sync is a sync which pulls only the data that has changed since the previous sync (as opposed to all the data available in the data source).
+
+This is especially important if there are a large number of records to sync and/or the API has tight request limits which makes a full sync of all records on a regular schedule too expensive or too slow.
+
+Incremental syncs are usually implemented using a cursor value (like a timestamp) that delineates which data was pulled and which data is new. A very common cursor value is an `updated_at` timestamp. This cursor means that records whose `updated_at` value is less than or equal than that cursor value have been synced already, and that the next sync should only export records whose `updated_at` value is greater than the cursor value.
+
+To use incremental syncs, the API endpoint needs to fullfil the following requirements:
+* Records contain a date/time field that defines when this record was last updated (the "cursor field")
+* It's possible to filter/request records by the cursor field
+
+To learn more about how different modes of incremental syncs, check out the [Incremental Sync - Append](/understanding-airbyte/connections/incremental-append/) and [Incremental Sync - Deduped History](/understanding-airbyte/connections/incremental-deduped-history) pages.
+
+## Configuration
+
+To configure incremental syncs for a stream in the connector builder, you have to specify how the records specify the **"last changed" / "updated at" timestamp**, the **initial time range** to fetch records for and **how to request records from a certain time range**.
+
+In the builder UI, these things are specified like this:
+* The "Cursor field" is the property in the record that defines the date and time when the record got changed. It's used to decide which records are synced already and which records are "new"
+* The "Datetime format" specifies the [format](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes) the cursor field is using to specify date and time,
+* The "Cursor granularity" is the smallest time unit supported by the API to specify the time range to request records for
```

Comment:
this should either have a link to some external docs or a description / enumeration of the values that are accepted. `PT1S` doesn't mean much without context